packages/stunner-editors/kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/expressions/jsinterop/props/Annotation.java

```diff
@@ -20,12 +20,11 @@
 
 @JsType
 public class Annotation {
-    public final String id;
+
     public final String name;
     public final Double width;
 
-    public Annotation(final String id, final String name, final Double width) {
-        this.id = id;
+    public Annotation(final String name, final Double width) {
```

Comment:
@tiagobento According to the DMN specs, Annotation (RuleAnnotationClause) doesn't store "id" field. (DMN Specs 8.3.2 - Table 35)
Thus, I removed it from the GWT Layer. 
In React world, that is still used here:
<img width="927" alt="Screenshot 2023-03-10 at 11 36 13" src="https://user-images.githubusercontent.com/16005046/224294380-363361a2-cdd0-481f-8090-12bb939e00a5.png">
Should we remove it from react layer too? How to substitute it in the above block of code? Thank you!

Reply:
I think it will be fine if you create a new UUID always, and use this as the accessor too.

Reply:
Done.